conf/script/build-ha-io

```diff
@@ -260,19 +260,19 @@ net_config_check() {
 # rvolume is mounted at /var/motr2.
 prepare_var_motr_dirs() {
     log "${FUNCNAME[0]}: make sure there are no stale mounts hanging..."
-    mountpoint -q /var/motr1 && sudo umount /var/motr1 || true
-    ssh $rnode 'mountpoint -q /var/motr2 && sudo umount /var/motr2 || true'
+    ssh $lnode "mountpoint -q /var/motr1 && sudo umount /var/motr1 || true"
```

Comment:
Well, I hope it will ssh it itself without a password.

Also, please don't use double-quotes when unnecessary (when you have nothing to evaluate inside the string). See https://github.com/bahamas10/bash-style-guide#quoting.

Reply:
Yes, local ssh is configured on the hardware, it works for other commands.
>don't use double-quotes when unnecessary (when you have nothing to evaluate inside the string)

okay.